components/checkbox/metadata/checkbox.yml

```diff
@@ -2,6 +2,10 @@ name: Checkbox
 status: Verified
 description: "Checkboxes allow users to select multiple items from a list of individual items, or mark one individual item as selected."
 SpectrumSiteSlug: https://spectrum.adobe.com/page/checkbox/
+sections:
+  - name: A note on emphasized and quiet
```

Comment:
Should this be a migration guide?

Reply:
Hmm, I don't know. I have just been writing "A note on emphasized and quiet, " but technically, they should _not_ change their checkboxes to emphasized where they were previously default, and their quiet will already look like the default ones looked before, so... Maybe it needs wording changes?

Reply:
@GarthDB I went ahead and made the section a `Migration Guide` and added instructions. Makes sense!